_overviews/macros/annotations.md

```diff
@@ -16,11 +16,12 @@ permalink: /overviews/macros/:title.html
 
 **Eugene Burmako**
 
-Macro annotations are only available with the macro paradise plugin (in Scala 2.10.x, 2.11.x and 2.12.x alike).
-Their inclusion in official Scala might happen in Scala 2.13, but there is no certainty about it yet.
-Follow the instructions at the ["Macro Paradise"](paradise.html) page to download and use our compiler plugin.
+Macro annotations are available in Scala 2.13 with the `-Ymacro-annotations` flag, but only with the macro paradise 
```

Comment:
The second part is a hard to parse for me. Suggestion: `, and with the macro paradise plugin from Scala 2.10.x to Scala 2.12.x`